README.md

```diff
@@ -4,11 +4,35 @@ Getting images from webcam by telegram bot in python3.
 
 If you need <i>proxy</i> - just uncomment line in botext.py and set valid proxy url.
 
+####Dependencies:
 
-To run and stop apps - use appropriate shell scripts.
+```apt install python3-pip python3-dev python3-opencv build-essential libssl-dev libffi-dev libgnutls28-dev```
 
+```pip3 install setuptools psutil proxybroker python-telegram-bot PySocks```
```

Comment:
it's common for python projects to list requirements in a separate file named "requirements.txt"